horace_core/sqw/axes_blocks/@AxesBlockBase/private/calc_bin_volume_.m

```diff
@@ -0,0 +1,34 @@
+function volume = calc_bin_volume_(~,ax_in_cell)
+%CALC_BIN_VOLUME_ calculate the volume of a lattice cell defined by the
+%cellarray of lattice axes.
```

Comment:
is it worth saying what the units are? Particularly if we are looking at hklE so the axes are heterogeneous

Reply:
no, whatever axes are, the units are. The axes for hkldE still homogeneous.